website/src/_posts/2019-05-27-looping-timers.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title:  "Looping timers in Apache Beam"
+date:   2019-05-27 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - Reza Rokni
```

Comment:
for the authors, you need to add a block for yourself in website/src/_data/authors.yml ... this section uses the block identifier to look up author info when it generates the page. So this should look something like this:

```
authors:
  - rezarokni (or whatever you name your block in authors.yml)
  - klk
```